src/arch/xtensa/lib/cpu.c

```diff
@@ -32,7 +32,6 @@ extern struct core_context *core_ctx_ptr[PLATFORM_CORE_COUNT];
 extern struct xtos_core_data *core_data_ptr[PLATFORM_CORE_COUNT];
 
 static uint32_t active_cores_mask;
-static int dsp_sref[PLATFORM_CORE_COUNT];	/**< simple ref counter */
```

Comment:
The commit message is missing a lot of detail. Why doe  this save time ? How does it save time. When are quick clock changes needed. Please give background.

Reply:
Commit message updated. Soon we will have dynamic clock switching and on TGL we need to power up core 3 if HPRO selected, so dynamic clock switching = dynamic core power up/down in this case